keps/sig-cluster-lifecycle/0003-cluster-api.md

```diff
@@ -48,16 +45,16 @@ last-updated: 2018-01-22
 
 ## Summary
 
-We are building a set of Kubernetes cluster management APIs to enable common cluster lifecycle operations (install, upgrade, repair, delete) across disparate environments. 
-We represent nodes and other infrastructure in Kubernetes-style APIs to enable higher level controllers to update the desired state of the cluster (e.g. the autoscaling controller requesting additional machines) and reconcile the world with that state (e.g. communicating with cloud providers to create or delete virtual machines). 
+We are building a set of Kubernetes cluster management APIs to enable common cluster lifecycle operations (install, upgrade, repair, delete) across disparate environments.
```

Comment:
can you move these whitespace changes to another PR or ask @cheftako to run a linting pass?